README.md

```diff
@@ -310,14 +310,73 @@ $ ninja -C out/Release
 
 ### Running tests
 
-Run:
+#### Build
+
+Build (includes tests):
 
 ```bash
 $ ./gyp_uv.py -f make
 $ make -C out
+```
+
+#### Run all tests
+
+```bash
 $ ./out/Debug/run-tests
 ```
 
+This driver forks and executes tests one by one.
+
+#### Run one test
+
+The list of all tests is in `test/test-list.h`.
+
+Driver forks and executes test in a child process:
+
+```bash
+$ ./out/Debug/run-tests TEST_NAME
+```
+
+Driver executes test in its own process:
+
+```bash
+$ ./out/Debug/run-tests TEST_NAME TEST_NAME
+```
+
+#### Debugging tools
+
+You might want to make use of debugging tools like gdb, valgrind, perf etc.
```

Comment:
Can you try to avoid "you" and other personal pronouns in technical documentation.